[2023-01-14T13:53:52.332+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2023-01-14T13:51:44.200723+00:00 [queued]>
[2023-01-14T13:53:52.384+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2023-01-14T13:51:44.200723+00:00 [queued]>
[2023-01-14T13:53:52.400+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-01-14T13:53:52.401+0000] {taskinstance.py:1284} INFO - Starting attempt 3 of 3
[2023-01-14T13:53:52.402+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-01-14T13:53:52.452+0000] {taskinstance.py:1304} INFO - Executing <Task(SqliteOperator): create_table> on 2023-01-14 13:51:44.200723+00:00
[2023-01-14T13:53:52.462+0000] {standard_task_runner.py:55} INFO - Started process 34498 to run task
[2023-01-14T13:53:52.471+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'manual__2023-01-14T13:51:44.200723+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpq_307umb']
[2023-01-14T13:53:52.476+0000] {standard_task_runner.py:83} INFO - Job 30: Subtask create_table
[2023-01-14T13:53:52.616+0000] {task_command.py:389} INFO - Running <TaskInstance: user_processing.create_table manual__2023-01-14T13:51:44.200723+00:00 [running]> on host 35e95982c619
[2023-01-14T13:53:52.796+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-01-14T13:51:44.200723+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-14T13:51:44.200723+00:00
[2023-01-14T13:53:52.806+0000] {sql.py:253} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS users (
              firstname TEXT NOT NULL,
              lastname TEXT NOT NULL,
              country TEXT NOT NULL,
              username TEXT NOT NULL,
              password TEXT NOT NULL,
              email TEXT NOT NULL PRIMARY KEY
            );
            
[2023-01-14T13:53:52.835+0000] {base.py:73} INFO - Using connection ID 'db_sqlite' for task execution.
[2023-01-14T13:53:53.531+0000] {base.py:73} INFO - Using connection ID 'db_sqlite' for task execution.
[2023-01-14T13:53:53.534+0000] {sql.py:364} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
              firstname TEXT NOT NULL,
              lastname TEXT NOT NULL,
              country TEXT NOT NULL,
              username TEXT NOT NULL,
              password TEXT NOT NULL,
              email TEXT NOT NULL PRIMARY KEY
            );
            , parameters: None
[2023-01-14T13:53:53.563+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230114T135144, start_date=20230114T135352, end_date=20230114T135353
[2023-01-14T13:53:53.628+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-01-14T13:53:53.732+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
